src/Features/Auth/epics.js

```diff
@@ -33,16 +38,23 @@ export const signUpEpic = (action$) =>
   action$.pipe(
     ofType(SIGNUP),
     mergeMap(({ payload }) => {
+      delete payload['contact'];
       return ajax({
         url: 'http://localhost:4000/v1/auth/register',
         method: 'POST',
         body: payload,
       }).pipe(
         mergeMap((res) => {
-          const {
-            tokens: { access },
-          } = res.response;
-          return of(loginSuccess(access));
+          const { user, tokens } = res.response;
+          const { name } = user;
```

Comment:
Why are you de structuring it in multiple lines ..
Can't we do it like below ??
`const {user: { name }, tokens: { refreshToken, access } }`